app/src/main/java/org/simple/clinic/bloodsugar/history/BloodSugarHistoryScreen.kt

```diff
@@ -59,7 +59,8 @@ class BloodSugarHistoryScreen : BaseScreen<
     ScreenBloodSugarHistoryBinding,
     BloodSugarHistoryScreenModel,
     BloodSugarHistoryScreenEvent,
-    BloodSugarHistoryScreenEffect>(), BloodSugarHistoryScreenUi, BloodSugarHistoryScreenUiActions {
+    BloodSugarHistoryScreenEffect,
+    Any>(), BloodSugarHistoryScreenUi, BloodSugarHistoryScreenUiActions {
```

Comment:
Instead of `Any`, won't `None` or `Unit` be a better option here since there are no effects?

Reply:
I tried using `Nothing`, but it showed some compile errors trying to infer type. Let me give `Unit` a try.

Reply:
Did you make this change? I don't see it. 🤔

Reply:
Forgot to make this change, pushed it now